justfile

```diff
@@ -10,7 +10,7 @@ dev:
 setup:
     node --version
     npm --version
-    npm install
+    npm install --legacy-peer-deps
```

Comment:
This is really annoying, and should be removed sometime soon when sveltekit version 2 and storybook and some bits of vite all agree on working together